helm/secrets-provider/values.schema.json

```diff
@@ -112,6 +112,16 @@
               "type": "string",
               "minLength": 1
             },
+            "timeout": {
+              "type": "number",
+              "minLength": 1,
+              "minimum": 5
```

Comment:
why minimum 5?
I would assume non-negative is enough, although in many places `timeout=-1` means forever, which is also valid.
What say you?
@oburstein-hub do you have something to add? should we allow 0 or even -1 as valid timeout values?

Reply:
@oburstein-hub ^

Reply:
it is not convinient to put timeout 0, although -1 is a common way to sign infinite timeout. 1 can be a minimal value